topology/graph/client.go

```diff
@@ -178,6 +187,8 @@ func NewAsyncClient(addr string, port int, path string) *AsyncClient {
 		Port:      port,
 		Path:      path,
 		messages:  make(chan string, 500),
+		quit:      make(chan struct{}, 1),
```

Comment:
That's what I did at first, but then I thought it would be strange to exit if we do `c.quit <- false`. Maybe it was just too much thinking :-)
Fixed